### ISSUE TITLE
Email keylogger logs.

### ISSUE THREAD

**Issue by bitterypaul** [MaxXor/xRAT#111]
A Feature involving sending keylogger logs to the specified email address.

**Comment by yankejustin** [comment]
Just to clarify, you want to redirect the logs gathered on the client so it will send to an e-mail instead of the server? It is a bit confusing because <b>so that they can be accessible even when client is turned off.</b> seems to imply that we can't currently access logs obtained from the client that was turned off. Once the client sends the logs, the client can be disconnected and we will still have them.

**Comment by DragonzMaster** [comment]
i agree with @yankejustin you can get the logs when ever you want as the client is running and connected to the server and they will be saved in "Logs" Folder so they are already accessible all the time
and the e-mail will need an internet connection and also the current keylogger so i see there is no difference
anyway thnx for your suggestion

**Comment by bitterypaul** [comment]
when the client cannot connect to the server(or when server is off),the client can email of the copy of logs to an email address.

**Comment by yankejustin** [comment]
Oh, I see... As an alternative! Hmmm..

**Comment by Abdullah2993** [comment]
Probably not a good Idea...as you have to embed the email/password in the client...a rather good approach would be to host some PHP scripts on web server so that the client can push logs to it.
